app/views/coronavirus/index.html.erb

```diff
@@ -46,6 +67,13 @@
           text: "Publish",
           margin_bottom: true
       } %>
+    <% end %>
+
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "6. Check live",
+      padding: true
+    } %>
+    <%= form_with(url: coronavirus_publish_path) do %>
```

Comment:
I think this is a duplicate line (the good one is on line 48) - the form needs to wrap the radio buttons and the publish button

Reply:
we can remove the form I think - It's just a link